src/backend/core/tests/teams/test_core_api_teams_create.py

```diff
@@ -48,3 +48,35 @@ def test_api_teams_create_authenticated():
     team = Team.objects.get()
     assert team.name == "my team"
     assert team.accesses.filter(role="owner", user=user).exists()
+
+
+def test_api_teams_create_authenticated_slugify_name():
+    """
+    Creating teams should automatically generate their slug.
+    """
+    identity = IdentityFactory()
+    jwt_token = OIDCToken.for_user(identity.user)
+
+    response = APIClient().post(
+        "/api/v1.0/teams/",
+        {
+            "name": "my team",
```

Comment:
Nice one ! I would parametrize this tests to cover more case, as strings containing special characters, double spaces, etc..

Reply:
Great idea ! 

Should I use param with tuples, such as [("my team", "my team"), ("my     team", "my team")] to make sure slugify is the expected str ?

Reply:
OK I did that and it works wonderfully. I love it, thanks for reminding me that existed ! 

I used the parameters on another test that the one you were pointing out : 
- `test_api_teams_create_authenticated_slugify_name` tests a slug is added 
- `test_api_teams_create_authenticated_expected_slug` tests all the aforementioned scenarios (previously unaccented scenario only)